README.md

```diff
@@ -6,13 +6,18 @@ A  ICS / ICal parser and serialiser for Golang.
 Because the other libraries didn't quite do what I needed.
 
 Usage, parsing:
-```
+```golang
     cal, err := ParseCalendar(strings.NewReader(input))
 
 ```
 
-Creating:
+Usage, parsing from a URL :
+```golang
+    cal, err := ParseCalendar("an-ics-url")
```

Comment:
Minor, but an example URL + a http client custom example might be nice. I've been contemplating making this more readable.